pyproject.toml

```diff
@@ -58,8 +58,6 @@ numpy = [
 ]
 matplotlib = ">=2.2.0"
 vrplib = "^1.4.0"

```

Comment:
Opting to keep `vrplib` as a dependency in `pyvrp`. I remember discussing having `vrplib` as extra, but it's a low-weight dependency and quite useful to have for quick benchmarking.